cmd/sniproxy/snipproxy.go

```diff
@@ -23,6 +25,11 @@ import (
 
 var ports = flag.String("ports", "443", "comma-separated list of ports to proxy")
 
+var (
+	tsMBox   = dnsmessage.MustNewName("support.tailscale.com.")
+	ip4, ip6 netip.Addr
```

Comment:
The use of these look like a data race. Let's just remove the caching.

Reply:
Removed caching, calls s.ts.TailscaleIPs()